<launch>
    <node name="extract" pkg="image_view" type="extract_images" respawn="false" required="true" output="screen" cwd="ROS_HOME">
        <remap from="image" to="/proc_img/image_raw"/>
    </node>
</launch>
